src/main/target/WORMFC/target.mk

```diff
@@ -0,0 +1,19 @@
+F405_TARGETS    += $(TARGET)
+FEATURES        += VCP
+
+TARGET_SRC = \
+			drivers/accgyro/accgyro_mpu.c \
+            drivers/accgyro/accgyro_spi_icm20689.c \
+            drivers/accgyro/accgyro_mpu6500.c \
+            drivers/accgyro/accgyro_spi_mpu6500.c \
+            drivers/barometer/barometer_lps.c \
+            drivers/max7456.c \
+            io/osd.c
+			
+            
+TARGET_SRC += \
```

Comment:
Please see my related comment at the SDIO driver PR. This could be added as an feature "SDIO" in the feature line on top of this file. Please also watch indentation.

Reply:
If we want to support this straight away sure why not